Oahu/OahuEvaluator.swift

```diff
@@ -1,11 +1,12 @@
 import Foundation
 
 open class OahuEvaluator: NSObject, Evaluator {
+
     open var closure: (_ urlIntercepted: String) -> Void
     open var url: String
     open var isAbsoluteUrl: Bool
 
-    public required init(url: String, isAbsoluteUrl: Bool, closure: @escaping (_ urlIntercepted: String) -> Void) {
+    public required init(url: String, isAbsoluteUrl: Bool, closure: @escaping (String) -> Void) {
```

Comment:
Why we could want to `escape` the closure arguments? I think it can be dangerous, because someone can change the value or even reference of the arguments. What do you think? By default the closure is `@nonescaping` and it's safe :) 

Reference at: https://swift.org/migration-guide/